app/models/survey_execution_state.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: survey_execution_states
+#
+#  id                                   :integer          not null, primary key
+#  question_id                          :integer
+#  respondent_id                        :integer
+#  created_at                           :datetime         not null
+#  updated_at                           :datetime         not null
+#
+
+class SurveyExecutionState < ActiveRecord::Base
```

Comment:
It maybe ambiguous with drafting a survey or state of a survey itself. When I hear `SurveyState`, I don't think of the survey execution like we do.

I'm open to name suggestions though and discussion @katrinacrisostomo